sparkler-api/src/test/resources/user-agents.txt

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# User agents to be used
```

Comment:
Yeah I guess in Nutch it was simply part of the Fetcher core, and I don't understand why a specific fetcher plugin would have a different way of using rotating agents such that it needed any rotating agent functionality in the plugin. Maybe we're saying the same thing. In addition I guess I don't understand why rotating agents (including the `user-agents.txt` file is part of this PR). Thanks TG.